packages/cli/src/commands/config/add.ts

```diff
@@ -35,6 +35,12 @@ export function Add(_: IContext): CommandModule<{}, Args> {
               .map((type) => `${name}.${type}`);
           })
           .flat(),
+      }).check(() => {
+        if (!Project.detectLanguage()) {
+          throw new Error('Invalid project. Directory should contain a package.json (typescript) or .sln (csharp) file.');
```

Comment:
Instead of "Invalid project" what are your thoughts on asking a question? Like "Are you sure you are in a valid directory?"

Reply:
fixed it to something along those lines. it's a warmer error message for sure.